docs/architecture/decisions/0024-production-release-cadence.md

```diff
@@ -0,0 +1,30 @@
+# 24. Role-based Access Control
+
+Date: 2023-11-02
+
+## Status
+
+In Review
+
+## Context
+
+Going into our first production launch we need a plan describing what our release cadence for both our staging and stable envirornments will be. Currently, we release to staging whenever there are significant changes made, but we have not been making releases to stable with the same frequency.
+
+## Considered Options
+
+**Option 1:** Releasing to stable/staging once a sprint
+Releasing once a sprint would mean that we release the past sprint's work to stable at the end of the current sprint. At the same point, the current sprint's work would be pushed to staging, thus making staging a full sprint ahead of stable. While this is more straight forward, it means our users would have to wait longer to see changes that weren't deemed critical.
+**Option 2:** Releasing to stable/staging once a week
+Releasing once a week would follow the same flow but with code being released to staging one week before the same code is released to stable. This would make stable only one week behind staging and would allow us to roll out minor bug fixes and faster with greater speed. The negative side is that we have less time to see if errors occur on staging
+
+In both of the above scenarios the release date would fall on the same day of the week that the sprint starts, which is currently a Wednesday. Additionally, in both scenarios the release commits would eventually be tagged with both a staging and stable tag. Furthermore, critical bugs or features would be exempt from these restrictions based on CISA's discretion.
+
+## Decision
+
+We decided to go with option 2 and release once a week once in production. This will allow us to give users features and bug fixes faster while still allowing enough time on staging for quality to be maintained.
+
+## Consequences
+
+Work not completed by end of the sprint will have to wait to be added to stable. Also, making hit fixes for bugs that are found on stable will be a little more complicated to fix.
```

Comment:
> making hit fixes

making _quick_ fixes?

Reply:
Can we say more about how we would make quick fixes? I think that we could make a branch with a special name and then cut a stable `vX.Y.Z` release pointing to that branch, but then we have to make sure to merge that quick fix branch back into main so that future releases to staging and stable also get the fix.

Reply:
@neilmb I put this in the readme, mostly because I felt odd placing it in an ADR. I was thinking the adr is what the decision is and ramifications, while the readme includes the nitty gritty details of how to do xyz. Do you still think I should copy over those instructions here?

Reply:
Sure, readme is good too.